tests/integration/main.rs

```diff
@@ -0,0 +1,48 @@
+mod artichoke;
+
+use std::env;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+#[allow(dead_code)]
+#[derive(Debug)]
+struct CommandOutput<'a> {
```

Comment:
What's your thoughts on a CommandRequest DAO sort of thing, and then doing something like `CommandOutput::from_request(request: &CommandRequest) -> Self` sort of thing too?

Reason, it'd be nice to store some of this in the CommandOutput (e.g. call args), since it might help understand how the snapshot was created etc (note, the snapshots don't auto clean up)